pypit/arparse.py

```diff
@@ -4417,3 +4418,60 @@ def parse_binning(binning):
         return 1,1
     else:
         return binspatial, binspectral
+
+
+def dummy_settings(pypitdir=None, nfile=10, spectrograph='shane_kast_blue',
+                   set_idx=True):
+    """ Generate settings classes
```

Comment:
Add more to this doc string.  Is this a Dummy settings class?
Is it mainly for testing?

Reply:
I just copied this directly from where it was before in arutils.py, docstring and all.  I don't know who originally wrote the function, but it's used in a number of the tests.  So, yes, I think it's mostly for testing.  So I slightly edited the docstring, but I would need the original author to flesh it out.